test/sql-tap/table.test.lua

```diff
@@ -495,7 +475,7 @@ test:do_catchsql_test(
           key int,
           "14_vac" int,
           fuzzy_dog_12 varchar(10),
-          beginn SCALAR,
+          begin_n SCALAR,
```

Comment:
What is the reason for this renaming?

Reply:
Checkpatch throws an error thinking the word is misspelled.

Reply:
Got it, thanks!